labellab-flask/api/models/User.py

```diff
@@ -22,11 +23,13 @@ class User(db.Model):
     projects = db.relationship('Project', 
                                backref='user', 
```

Comment:
can you change backref into owner for more meaningful, project has owner right as the ER?

Reply:
I mean the backref has to be the tablename of the model it is back-populating in. so I cannot change the name of the backref here 😅. In any case i have named the user.id column as admin_id in projects table so that will reduce the confusion.

Reply:
ok then :slightly_smiling_face: